examples/README.md

```diff
@@ -1,4 +1,184 @@
+      kind: CustomResourceDefinition
+      name: workloads.workload.example.io
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+```
+</details>
+
+```bash
+kubectl --kubeconfig $HOME/.kube/karmada.config --context karmada-apiserver apply -f workload-crd-cpp.yaml
+```
+
+#### Step2: Deploy webhook configuration in karmada-apiserver
+
+Execute below script:
+
+webhook-configuration.sh
+
+<details>
+
+<summary>unfold me to see the script</summary>
+
+```bash
+#!/usr/bin/env bash
+
+export ca_string=$(cat ${HOME}/.karmada/server-ca.crt | base64 | tr "\n" " "|sed s/[[:space:]]//g)
+export temp_path=$(mktemp -d)
+
+cp -rf "examples/customresourceinterpreter/webhook-configuration.yaml" "${temp_path}/temp.yaml"
+sed -i'' -e "s/{{caBundle}}/${ca_string}/g" "${temp_path}/temp.yaml"
+kubectl apply -f "${temp_path}/temp.yaml"
+rm -rf "${temp_path}"
+```
+
+</details>
+
+```bash
+chmod +x webhook-configuration.sh
+
+./webhook-configuration.sh
+```
+
+#### Step3: Deploy interpreter webhook example in karmada-host
+
+```bash
+kubectl --kubeconfig $HOME/.kube/karmada.config --context karmada-host apply -f examples/customresourceinterpreter/karmada-interpreter-webhook-example.yaml
+```
+
+### Usage
+
+Create a `Workload` resource and a `PropagationPolicy`:
+
+workload-interpret-test.yaml:
+
+<details>
+
+<summary>unfold me to see the yaml</summary>
+
+```yaml
+apiVersion: workload.example.io/v1alpha1
+kind: Workload
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+spec:
+  replicas: 3
+  paused: false
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      containers:
+      - image: nginx
+        name: nginx
+---
+apiVersion: policy.karmada.io/v1alpha1
+kind: PropagationPolicy
+metadata:
+  name: nginx-workload-propagation
+spec:
+  resourceSelectors:
+    - apiVersion: workload.example.io/v1alpha1
+      kind: Workload
+      name: nginx
+  placement:
+    clusterAffinity:
+      clusterNames:
+        - member1
+        - member2
+        - member3
+    replicaScheduling:
+      replicaDivisionPreference: Weighted
+      replicaSchedulingType: Divided
+      weightPreference:
+        staticWeightList:
+          - targetCluster:
+              clusterNames:
+                - member1
+            weight: 1
+          - targetCluster:
+              clusterNames:
+                - member2
+            weight: 1
+          - targetCluster:
+              clusterNames:
+                - member3
+            weight: 1
+```
+
+</details>
+
+```bash
+kubectl --kubeconfig $HOME/.kube/karmada.config --context karmada-apiserver apply -f workload-interpret-test.yaml
+```
+
+#### InterpretReplica
+
+You can get `ResourceBinding` to check if the `replicas` field is interpreted successfully.
+
+```bash
+kubectl get rb nginx-workload -o yaml 
+```
+
+#### ReviseReplica
+
+You can get `ResourceBinding` to check if the replicas field is revised successfully.
```

Comment:
The `ReviseReplica` operation doesn't work on ResourceBinding, it will revise replicas of workload in `Work` object:

https://github.com/karmada-io/karmada/blob/7323eadb18bf928e5bb5a44a190837bad44349dc/pkg/controllers/binding/common.go#L106

Reply:
Thanks for your reviews, I will do them.